### PR TITLE
fixed grammar issue on about page

### PR DESCRIPTION
**Pull request by jossters**
# Description
Screen Shot 2021-08-17 at 9 33 32 PM
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue

## Change Status

- [x] Complete, tested, ready to review and merge

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts
- [ ]

### CODE REVIEW

**Review by JAaron93 — APPROVED** [review]
I like what you did here. 

- Clear and concise language
- Updated the length of time we've been developing this from six to seven months
- Capitalized Twitter 
- Removed mentions of Reddit
- Corrected typos

Only nitpicky thing I'd change is the last sentence:
', that have ever contributed to this project.'

Otherwise it's pretty much perfect. Might add some more details next week about the bot once our team has actually completed it.

File: src/components/about/About.js
Comment (alvin-joseph) [inline src/components/about/About.js:152]: Nice work updating this text! It definitely needed it! I agree with Joshua about the last sentence though. I would maybe change it to "Below are all of the Lambda School students that have worked on this website, including their GitHub profiles."

File: src/components/about/About.js
Comment (RyanFikejs) [inline src/components/about/About.js:15]: Avoiding the whole "data is" vs. "data are" debacle. Clever wording!

File: src/components/about/About.js
Comment (RyanFikejs) [inline src/components/about/About.js:40]: I'm re-reading this thing multiple times to be sure: I think this was actually ok as "contextualizes" vs. "contextualize".

File: src/components/about/About.js
Comment (RyanFikejs) [inline src/components/about/About.js]: Consider adding that the specific NLP model used here is a tailored version of BERT. I also suggest including a link to some information about BERT: https://en.wikipedia.org/wiki/BERT_(language_model)

File: src/components/about/About.js
Comment (RyanFikejs) [inline src/components/about/About.js]: Suggesting we change to:
"Our BERT model categorizes each incident report we receive."

File: src/components/about/About.js
Comment (RyanFikejs) [inline src/components/about/About.js]: Technically the incident report exists in our database already and its status is updated to 'approved' or 'rejected'. By keeping both in the database we are able to be sure that our twitter scraper doesn't pull in a duplicated instance in our database.

File: src/components/about/About.js
Comment (RyanFikejs) [inline src/components/about/About.js:106]: Thank you for including an Oxford comma. I know that they're considered "optional" in conventional, American English, but I still find them very useful.

File: src/components/about/About.js
Comment (devin-mitchell) [inline src/components/about/About.js]: very nit picky, but could maybe change this sentence to something like "...reports regarding incidents of police..." just to have a bit more flow?  Just a thought, feel free to do whatever you want.

**Review by devin-mitchell — APPROVED** [review]
Looks great in my opinion.  Left one very small note to possibly consider.

File: src/components/about/About.js
Comment (devin-mitchell) [reply src/components/about/About.js:152]: I agree with Alvin and Joshua.  Little easier to read/understand.

File: src/components/about/About.js
Comment (jossters) [reply src/components/about/About.js]: Yeah, I agree it sounds better thanks.

File: src/components/about/About.js
Comment (RyanFikejs) [inline src/components/about/About.js]: We need to establish a differentiation here between our Twitter scraper and our Twitter bot. The scraper is responsible for gathering tweets with content specific criteria including (but not limited to) key words such "police" and "misconduct" and their synonyms. It does this multiple times a day (I think every 4 hours) and checks the incoming data against data in our database to prevent duplicates being entered into the database and then passes them to the BERT model to be classified and await administrator approval.

The Twitter bot is a tool that we are currently building that is able to reach out to Twitter users to confirm or supply missing information. It is also triggered by the Administration team and new data gathered awaits approval before being marked as an 'approved' incident report.

Comment (RyanFikejs) [reply src/components/about/About.js]: (I think that the previous Labs cohorts had not fully made this distinction clear in the about section because they had not yet reached that stage of development.)

File: src/components/about/About.js
Comment (jossters) [reply src/components/about/About.js:152]: How about 'all the' instead of 'all of the'.

File: src/components/about/About.js
Comment (RyanFikejs) [inline src/components/about/About.js]: Suggesting:
"Below are the students who have worked on this project with links to their Github profiles included.

**Review by RyanFikejs — CHANGES_REQUESTED** [review]

File: src/components/about/About.js
Comment (alvin-joseph) [reply src/components/about/About.js:40]: Yeah I agree. I kept second guessing myself when reading that lol

**Comment by RyanFikejs** [comment]
Johnny, thank you for tackling this very necessary sweep of the about section. Don't let the number of my comments dissuade you, you're doing good work, we just have a few things to address before merging. If you have any questions and I don't appear to be responsive here, please feel free to DM me on slack for more clarification. (I will try to check here first.)

File: src/components/about/About.js
Comment (alvin-joseph) [reply src/components/about/About.js:152]: Yeah that sounds better actually

File: src/components/about/About.js
Comment (jossters) [reply src/components/about/About.js]: I agree, ill include BERT.

File: src/components/about/About.js
Comment (jossters) [reply src/components/about/About.js]: I'm using this to update the section 'Our Twitter Scrapper/Bot'

**Review by devbrunopaula — COMMENTED** [review]
Great changes 👏

File: src/components/about/About.js
Comment (jossters) [reply src/components/about/About.js]: I also used this in the 'confirm' section.

Comment (RyanFikejs) [reply src/components/about/About.js]: Good thinking!

File: src/components/about/About.js
Comment (RyanFikejs) [inline src/components/about/About.js]: I was going to say to just change "scrapper" to "scraper" and I think we're looking good, but I went ahead and committed that change on my own.

**Review by RyanFikejs — APPROVED** [review]
Thanks for taking the time to go over the changes again Johnny! I feel like this work is sometimes thankless and never ending, so thanks for getting it done!

File: src/components/about/About.js
Comment (rhiag) [reply src/components/about/About.js:152]: This some really great work. Very detailed yet easy to understand!

**Pull request merged**
